### DISCUSSION TITLE
HIP-0014: Hard fork to extend name claim period

### DISCUSSION THREAD

**Discussion by NetOpWibby** [handshake-org/HIPs#57] [Polls]
Technical overview: https://hsd-dev.org/HIPs/proposals/0014/
High-level overview: https://twitter.com/DotTheShake/status/1613893968185507840

To paraphrase The Shake's explanations:
- Option 1: Reserve existing DNS TLDs like `.com`, `.net`, and `.org`.
- Option 2:  Reserve Alexa Top 100 names like `.reddit`, `.google`, and `.wikipedia`.
- Option 3: Reserve names of orgs from the initial snapshot like `.brave`, `.keybase`, and `.tor`.
- Option 4: Scorched Earth, Wild West, whatever you want to call it. Let everything expire, YOLO.

## Notes
- Amazon shut down their Alexa ranking system so any mention of an Alexa list is dated and does not bring into account the websites that have been launched since that shutdown and grown to popularity.
- `.google` is an existing ICANN TLD
- Brave and Keybase claimed their TLDs.

## Questions
- Companies are dropping their TLDs on a regular basis. How do we account for that? Periodic hard-forks to allow those names to be claimable? [Stellantis dropped](https://domainnamewire.com/2023/01/27/maserati-gone-car-company-cancels-five-top-level-domains/) `.abarth`, `.alfaromeo`, `.fiat`, `.maserati`, and `.lancia`. Of those, `fiat/` would be a premium name on Handshake.
- When do we conclude the results of this poll? Conversations about this hard-fork is spread across Telegram groups and Discord servers.

**Comment by enginux** [comment]
Reserve Option 1 and 2. The rest throw it to the wild.

**Comment by Falci** [comment]
What about rewards for name claims and developers claims?

**Reply by NetOpWibby** [reply]
Wouldn't those be intact as long as reservation is active?

**Reply by rithvikvibhu** [reply]
There were some ideas about removing the $hns reward but keeping the names claimable. This at least stops drop-catchers from buying up expiring domains and then claiming the 503 HNS.

I think this can be a separate poll (but part of the same hard fork). It fits with options 1,2,3 above.

**Reply by rithvikvibhu** [reply]
Not just removing, someone (@Falci?) suggested an experiential decay over time.

**Reply by NetOpWibby** [reply]
I like the experiential decay

**Reply by dotdwebo** [reply]
Another potential hard fork to consider.
Departing from the original purpose of the coins created.

1. Rewards for miners.
2. Tool to renew TLDs.
3. Tool to participate in the auction.
4. Tools to buy domains in the secondary market (shakedex and namebase).
5. Tool to speculate (to be honest, in every disruptive tech ecosystem, there will always be some speculators).


![venn](https://user-images.githubusercontent.com/100784480/215309679-1278faad-b574-4ef1-8f7b-e4ed33743728.png)


Apart from the TLD, the availability of coins on the market also has a significant effect on the sustainability of the project. Imagine a Venn diagram, where the first circle is the technology and the second circle is economics. Both intersect each other. Handshake and other blockchain technologies are at that intersection.

The longer the coins are unclaimed, the greater the chance of liquidity in economic transactions in the ecosystem being hampered. (Another hidden risk: There is a greater chance of large coin holders dumping their coins at unexpected times when ecosystem participants are not prepared to handle them). 

If the risk is still an isolated risk, maybe we can ignore it. Isolated risk means a risk whose impact is not too significant, easy to recover, and does not require a lot of time and energy to return to normal.

Systemic risk can be interpreted as a risk whose impact is too significant so that the chance for recovery is close to zero. (Example: Luna's Crash).

**Comment by galetaire** [comment]
I believe "root" names should be extended without expiration date. Handshake value does not come from ICANN current TLDs, so respecting the main root zone as much as possible is positive for the broader web. Future ICANN registrations will collide, well, such is life, they will be born in a new environment, so owners should have to consider buying also their Handshake counterpart. Same as big companies have to buy multiple SLDs currently (.com, .net, .org...) to avoid users confusion.

For all the other names, I feel good with any outcome. Anyway, I have voted for extending Top100 names.

**Reply by NetOpWibby** [reply]
I'm also a fan of not letting existing ICANN TLDs expire, ever.

**Reply by Falci** [reply]
The argument against making legacy TLD not expire is the urgency.
Legacy TLD owner won't need to onboard on Handshake, because there will be no risk of losing their names.
If those TLD also expire, they must to onboard or lose the names.

Not saying this is a good argument, just an argument.

**Comment by nglabs42** [comment]
I vote 1 and 2, and also agree on option 1 never expiring.

**Comment by dotdwebo** [comment]
1. Is there a website that provides information about TLDs that have been claimed or have not been claimed?

2. Is the faster a TLD claimed to have a significant impact on mass adoption? If so, what are the reasons why some entities haven't claimed their reserved TLD yet? (Has there been any research explaining why they are delaying claims?)

3. Would four years of extended claim duration increase some people's chances of claiming their TLD?

4. What can participants in the community/ecosystem do to increase the chances of multiple names (TLD) being claimed? Is there a need for a strategy, tactic, or template to invite, influence, or educate some reserved TLD owners to claim their TLD immediately?

**Reply by rithvikvibhu** [reply]
>1. Is there a website that provides information about TLDs that have been claimed or have not been claimed?

https://handshake.rsvp/ by @eskimo. Can check both claimed and unclaimed lists.

>2. If so, what are the reasons why some entities haven't claimed their reserved TLD yet?

Assuming you're asking about ICANN TLDs' registries, I can only guess with no backing research:
- claiming a name as a TLD may look like they are supporting a project
- legal implications with ICANN
- they're just waiting for other registries to claim first (~4 TLDs have claimed their names already)
- they just don't care or think it's not worth the time/$

>3. Would four years of extended claim duration increase some people's chances of claiming their TLD?

Again, assuming this is about ICANN TLDs, yes. We had some claims last year, I think we'll have more this year too. But either way, I think it's better to keep those TLDs reserved forever (claimable or not). This decision can be taken later after another 4 years though.

>4. Is there a need for a strategy, tactic, or template to invite, influence, or educate some reserved TLD owners to claim their TLD immediately?

Yes! Some people who have been doing outreach work can maybe share their structure and templates.

**Comment by hnshnshnshns** [comment]
Every quarter, I help dozens of people at Namepros 4rum claim their domains, which leads to a bunch of web2 domain hunters getting acquainted with HNS. *so don't wonder if there's a wallet that claims 5-20 domains.*. The number of HNS for each domain is too low compared to the value the project can receive. 100k Alexa , 95% have profitable business models, the companies behind them are most of the power of the internet today, and it only takes a few big brands to claim their domain, snow ball effect will happen.

**Reply by hnshnshnshns** [reply]
the guys at Namepros community mainly keep premium domains, not domains in 100k Alexa - the website is running. But it will bring in domain experts who make a living out of the domain name business. with HNS. I don't think the crowd out there cares much about buying and selling domains, it's enough that HNS needs to be easy to use with them.

**Reply by hnshnshnshns** [reply]
most of my clients lost money, some tried to buy back dozens of premium domains to claim 503 hns/1 domain, and they lost -500% profit. even the reward for the top 100k Alexa should be increased, because most of the brands in 100k Alexa have hefty profits, and they don't care about 503 HNS. That might be a fee for them to get used to HNS, oh well. but it's not a monetary reward for them.

**Comment by adriannaarlie** [comment]
Reserve Option 1 and 2 . thanks

**Comment by osakagm** [comment]
HIP-0014: Important reserved names (about 1,000) stay reserved for 4 additional years, a total of 8.
The rest of the reserved names (about 89,000) become available for auctions after 4 years (no change)  ?
claim 89000 domains is even more important than claiming 1000 icann TLD. but basically we should agree on both.

**Comment by jacobjonz** [comment]
Yes, I would say, extend the reservation for four more years for all domains as of now, but let the rewards expire.
It's evident that the remaining name owners could not be enticed by the rewards (or we were not able to reach out to them).

Not against a gradual decay of claimable rewards as mentioned above either, but not sure how much it would help.

The expired rewards should either go to the dev/marketing funds (possibly the one initiated by HandshakeJesus - not sure what his current name is) or else burned.

**Reply by hnshnshnshns** [reply]
hns jeus idea is a pretty centralized idea. HNS Jenus or Dweb Foundation is good but they are just one or two of the entities that support HNS.

**Comment by hnshnshnshns** [comment]
there are many times Chjango wants 100k Alexa to be reallocated to HNS development fund and can turn it into WHNS, I object to this. it's something pretty centralized, if someone wants to do WHNS they should buy HNS to do.
Too many people covet the names in 100k Alexa. 
it should be. there are even 100x Alexa companies whose value is many times greater than the current HNS Marketcap. So I hope we can cancel the HNS airdrop for developers, and pass it on to the reward for 100k Alexa, or better still, stay the same, and extend the time.

**Reply by jacobjonz** [reply]
Doesn't have to be them. If there is any way to do handle  marketing/dev funds decentralized, that works too..

At this point, there is no point in rewarding anyone beyond the first 1000 players. They had their chance to claim. Now it is just people doing drop catching.

**Comment by NetOpWibby** [comment]
Huge convo on Twitter at the moment: https://twitter.com/woodburn_nathan/status/1661324622602862592

Also, aox of Namebase mentioned in Discord that the name expiration stuff can be a **soft fork** instead.

**Comment by hnshnshnshns** [comment]
I have some changes in my opinion, I agree to cancel Option 2,3,4, and keep only option 1 .
and please can you cancel github rewards, instead of turning it into WHNS or something, burn them.
